controllers/usersControllers.js

```diff
@@ -69,13 +70,16 @@ const signIn = async (req, res, next) => {
   });
 };
 
-const signOut = async (req, res, next) => {
+const signOut = async (req, res) => {
   const { _id } = req.user;
-  await usersService.updateUser({ _id }, { token: null });
+  await usersServices.updateUser({ _id }, { token: null });
   res.status(204).json();
 };
 
-const getCurrent = (req, res) => {};
+const getCurrent = (req, res) => {
+  const { name, email, avatarURL, followers, following } = req.user;
+  res.status(200).json({ name, email, avatarURL, followers, following });
```

Comment:
I see the current is not ready yet.
You don't have to set the status to 200, this status is used by default